isolate.sh

```diff
@@ -762,46 +771,30 @@ function cleanup_secretshare() {
     cleanup_deployment "secretshare" $MASTER_NS
 }
 
-function check_if_certmanager_deployed() {
+function check_if_certmanager_requested() {
     local namespaces=$@
-    info "Checking for cert manager deployed in scope."
-    local deployed="false"
+    info "Checking for cert manager requested in scope."
+    local requested="false"
     for ns in $namespaces
     do
         opreqs=$(${OC} get opreq -n $ns --no-headers | awk '{print $1}' | tr '\n' ' ')
         for opreq in $opreqs
         do
             local return_value=$(${OC} get opreq $opreq -n $ns -o yaml | ${YQ} '.spec.requests[]' | grep "name: ibm-cert-manager-operator" || echo "fail")
             if [[ $return_value != "fail" ]]; then
-                deployed="true"
-                info "Cert manager requested in scope, moving on..."
+                requested="true"
+                info "Cert manager requested in scope"
                 break
             fi
         done
     done
 
-    if [[ $deployed == "false" ]]; then
-        info "Cert manager not requested in scope, deploying..."
-        cat <<EOF > /tmp/tmp-opreq.yaml
-apiVersion: operator.ibm.com/v1alpha1
-kind: OperandRequest
-metadata:
-  labels:
-    app.kubernetes.io/instance: operand-deployment-lifecycle-manager
-    app.kubernetes.io/managed-by: operand-deployment-lifecycle-manager
-    app.kubernetes.io/name: odlm
-  name: ibm-cert-manager-operator
-  namespace: $MASTER_NS
-spec:
-  requests:
-    - operands:
-        - name: ibm-cert-manager-operator
-      registry: common-service
-      registryNamespace: $MASTER_NS
-EOF
-
-    ${OC} apply -f /tmp/tmp-opreq.yaml
-    rm -f /tmp/tmp-opreq.yaml
```

Comment:
Also, at the end of isolation where we `wait_for_certmanager`.
- If `REQUEST_CERTMANAGER==true`, we will need to `wait_for_certmanager`.
- Inside `wait_for_certmanager`, we will still need to check if `cert manager requested in new scope ibm-common-services,cp2`. If it is not requested, then we will create an OperandRequest, to have ODLM install cert manager.